### ISSUE TITLE
Can't resolve '../../StyledIconBase'

### ISSUE THREAD

**Issue by Betree** [jacobwgillespie/styled-icons#614]
I tried to upgrade from 6.4.0 to 7.1.0 today and hit the following error:

```
> Failed to build
{ Error: (client) ../node_modules/styled-icons/boxicons-regular/LoaderAlt/LoaderAlt.esm.js
Module not found: Error: Can't resolve '../../StyledIconBase' in '/home/circleci/project/node_modules/styled-icons/boxicons-regular/LoaderAlt'
 @ ../node_modules/styled-icons/boxicons-regular/LoaderAlt/LoaderAlt.esm.js 3:0-54 8:32-46
 @ ./components/StyledSpinner.js
 @ ./components/StepsProgress.js
 @ ./pages/createOrderNewFlow.js
 @ multi ./pages/createOrderNewFlow.js
    at /home/circleci/project/node_modules/next/dist/build/index.js:144:31
    at finalCallback (/home/circleci/project/node_modules/webpack/lib/MultiCompiler.js:247:12)
    at runWithDependencies.err (/home/circleci/project/node_modules/webpack/lib/MultiCompiler.js:270:6)
    at done (/home/circleci/project/node_modules/neo-async/async.js:2920:13)
    at runCompilers (/home/circleci/project/node_modules/webpack/lib/MultiCompiler.js:174:48)
    at err (/home/circleci/project/node_modules/webpack/lib/MultiCompiler.js:181:7)
    at compiler.run (/home/circleci/project/node_modules/webpack/lib/MultiCompiler.js:263:7)
    at finalCallback (/home/circleci/project/node_modules/webpack/lib/Compiler.js:204:39)
    at hooks.done.callAsync.err (/home/circleci/project/node_modules/webpack/lib/Compiler.js:220:13)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/circleci/project/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:33:1)
    at AsyncSeriesHook.lazyCompileHook (/home/circleci/project/node_modules/tapable/lib/Hook.js:154:20)
    at onCompiled (/home/circleci/project/node_modules/webpack/lib/Compiler.js:218:21)
    at hooks.afterCompile.callAsync.err (/home/circleci/project/node_modules/webpack/lib/Compiler.js:547:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/circleci/project/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:15:1)
    at AsyncSeriesHook.lazyCompileHook (/home/circleci/project/node_modules/tapable/lib/Hook.js:154:20)
    at compilation.seal.err (/home/circleci/project/node_modules/webpack/lib/Compiler.js:544:30)
```

I guess it comes from https://github.com/jacobwgillespie/styled-icons/pull/612, probably because we don't use Typescript? I've downgraded to 7.0.0 and it works correctly.

Otherwise thanks for the automatic support of CommonJS and ES Modules code, that's a great change for us!

**Comment by jacobwgillespie** [comment]
Thanks for catching this - not TypeScript, just a misconfiguration on my part that omitted that file from the NPM package. 😞 I really need to get integration tests running that use the `.tgz` from `npm pack` to catch stuff like this.

It's building now and should be fixed in `v7.1.1`.  It does speed up TypeScript type-checking quite a bit, but you may also see some frontend performance improvements (reduced memory probably) when rendering multiple icons, since they share most of the heavy logic inside a common component now.

**Comment by jacobwgillespie** [comment]
This has been fixed in `v7.1.1` - sorry for the disruption!
